README.md

```diff
@@ -234,6 +234,7 @@ Keys supported by image output:
 * `dangling-name-prefix=[value]`: name image with `prefix@<digest>` , used for anonymous images
 * `name-canonical=true`: add additional canonical name `name@<digest>`
 * `compression=[uncompressed,gzip,estargz,zstd]`: choose compression type for layers newly created and cached, gzip is default value. estargz should be used with `oci-mediatypes=true`.
+* `compression-level=[value]`: compression level for gzip, estargz (0-9) and zstd (0-22)
```

Comment:
Could you add the default value?

Reply:
There is no universal default value. `-1` is the default in gzip.

Reply:
Actually, I see I made -1 match default in`zstd` as well(although non-spec). But estagz might still want a different default https://github.com/moby/buildkit/pull/2591#discussion_r796360476